src/Tests/Moryx.Communication.Sockets.IntegrationTests/HeaderProtocol/HeaderedCommunicationSocketTests.cs

```diff
@@ -10,6 +10,10 @@
 
 namespace Moryx.Communication.Sockets.IntegrationTests
 {
+    // preprocessor statement for differentiating between different os
+    // see: https://github.com/PHOENIXCONTACT/MORYX-Framework/issues/100 (Moryx-Framework/src/Moryx/Communication/Sockets/TcpTransmission.cs in line 85)
+    // only run tests if os is windows (constant _WINDOWS defined in property group in csproj-file)
```

Comment:
You can neglect the last sentence, since the csproj file is the default place for that 😊